astropy/time/tests/test_precision.py

```diff
@@ -500,7 +513,7 @@ def test_jd_add_subtract_round_trip(scale, jds, delta):
 
 @given(scale=sampled_from(STANDARD_TIME_SCALES),
        jds=reasonable_jd(),
-       delta=floats(-3*tiny, 3*tiny))
+       delta=floats(-0.5, 0.5).filter(lambda x: x == 0 or (abs(x) > (3 * tiny))))
```

Comment:
This reverses the goal of the tests... Sadly, the log of the failing run seems to have disappeared. Is there an example of a failure?

Reply:
Again, I will remove so you see example.